### PR TITLE
Add support to fields.go for protoreflect.EnumNumber with tests

### LINKED ISSUE #6: Support `protoreflect.EnumNumber` field type
We need to support the `protoreflect.EnumNumber` in our [processor field](https://github.com/faunists/deal-go/blob/main/processors/fields.go#L17) function that formats the message to its string representation!

- [ ] Make the function works correctly with`protoreflect.EnumNumber`
- [ ] Write tests to stress as many possibilities as possible

### PR DESCRIPTION
**Pull request by florkbr**
Resolves https://github.com/faunists/deal-go/issues/6

@thepabloaguilar please review!

### CODE REVIEW

**Comment by thepabloaguilar** [comment]
Hey @florkbr I think I created that issue with the wrong protoreflect type, but I really appreciate your contribution. I won't merge now until I'm sure what we need with Enums. Your contribution will not be for nothing, I'm going to tag this PR with `hacktoberfest-accepted` so it'll count for you as you can see in the image below:
image

**Comment by thepabloaguilar** [comment]
Hey @florkbr I think I created that issue with the wrong protoreflect type, but I really appreciate your contribution. I won't merge now until I'm sure what we need with Enums. Your contribution will not be for nothing, I'm going to tag this PR with `hacktoberfest-accepted` so it'll count for you as you can see in the image below:
image

File: processors/fields.go [hunk 1]
```diff
@@ -28,7 +28,9 @@ func FormatFieldValue(value protoreflect.Value) string {
 		return fmt.Sprintf("%q", v)
 	case []byte:
 		return fmt.Sprintf("%#v", v)
-	case protoreflect.Message, protoreflect.List, protoreflect.Map, protoreflect.EnumNumber:
+	case protoreflect.EnumNumber:
+		return fmt.Sprintf("%d", v)
+	case protoreflect.Message, protoreflect.List, protoreflect.Map:
```
Comment (thepabloaguilar) [inline processors/fields.go:32]: Please, remove `protoreflect.EnumNumber` from the `TODO` list in beginning of the function!

**Review by thepabloaguilar — CHANGES_REQUESTED** [review]
Ok, I've made some tests and your solution work as we need proposed in the original issue! Thanks

It has just a minor comment, fix that and we'll ready to go!

**Comment by florkbr** [comment]
@thepabloaguilar no worries! Just exploring what projects are out there and found yours filtering by Go!

Fixed the comment as requested. Let me know if you need anything else!

**Comment by florkbr** [comment]
@thepabloaguilar bump :)

**Comment by thepabloaguilar** [comment]
@florkbr the lint check failed!

**Comment by florkbr** [comment]
@thepabloaguilar sorry! Fixed!

**Comment by florkbr** [comment]
@thepabloaguilar is the current coverage I've added acceptable? Or do I need to add more?

**Comment by thepabloaguilar** [comment]
It's acceptable, in fact, `codecov/patch` failed because we don't have any test for the `default` case and you changed that to remove the `protoreflect.EnumNumber`!

I'll merge it tonight

**Review by thepabloaguilar — APPROVED** [review]
Thanks for you contribution @florkbr!!

**Pull request merged**